torch/csrc/jit/tensorexpr/schedule.cpp

```diff
@@ -766,14 +769,18 @@ SplitAxisWithMask::SplitAxisWithMask(
   CHECK(factor_on_inner) << "only factor_on_inner = True is supported for now";
 
   // TODO: Support dynamic shapes
-  int size = this->stop() - this->start();
-  if (size % factor != 0) {
-    CHECK(this->start() == 0) << "Non-zero start is not implemented yet";
-    if (this->stop() % factor != 0) {
+  Expr sizeExpr = this->stop() - this->start();
```

Comment:
This indentation for this block is messed up. Right?

Reply:
I don't think so?  What looks wrong about it?